developer-docs/upgrading_kubernetes.md

```diff
@@ -1,10 +1,11 @@
 # Upgrade Kubernetes Process
 
-From time to time we need to update the version of Kubernetes used by RKE2. This document serves as a how-to for that process and constitutes a "release". The following steps are laid out in order.
+From time to time, we need to update the version of Kubernetes used by RKE2. This document serves as a how-to for said process and elaborates on what constitutes a "release". Although the steps described in the document are laid out in order, the overall flow looks this way:
 
-This process will need to be done any time a new release is needed.
+<!-- Source: https://gist.github.com/jossemargt-3pillar/4c93f23821a8cf9558692002d10c70ef -->
```

Comment:
Is this line necessary?

Reply:
No, it came from the initial iterations. Let me remove it.